app/assets/stylesheets/home/components/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$text-color: #0077B6;
+$breakpoint-desktop: 992px;
+
+@mixin desactivated_navbarMobile{
+  .navbar{
```

Comment:
Most styling guides suggest to **always** leave a whitespace between a selector and a curly bracket. What do you think about adding it here?

Reply:
Yes, i'll do it!
Thank u  for the recommendation!

Reply:
About the bin / * files, I think they were added because I had just cloned the repository and had to install webpack, bundle install, create the database and migrate it ...
I want to assume that was it ... 🤔🤔